MobiFlight/Joysticks/HoneycombBravo.cs

```diff
@@ -43,39 +107,37 @@ protected override void SendData(byte[] data)
         protected override void EnumerateOutputDevices()
         {
             base.EnumerateOutputDevices();
-            Lights.Add(new JoystickOutputDevice() { Label = "AP Mode - HDG",                Name = "AP.hdg",                Byte = 1, Bit = 0 });
-            Lights.Add(new JoystickOutputDevice() { Label = "AP Mode - NAV",                Name = "AP.nav",                Byte = 1, Bit = 1 });
-            Lights.Add(new JoystickOutputDevice() { Label = "AP Mode - APR",                Name = "AP.apr",                Byte = 1, Bit = 2 });
-            Lights.Add(new JoystickOutputDevice() { Label = "AP Mode - REV",                Name = "AP.rev",                Byte = 1, Bit = 3 });
-            Lights.Add(new JoystickOutputDevice() { Label = "AP Mode - ALT",                Name = "AP.alt",                Byte = 1, Bit = 4 });
-            Lights.Add(new JoystickOutputDevice() { Label = "AP Mode - VS",                 Name = "AP.vs",                 Byte = 1, Bit = 5 });
-            Lights.Add(new JoystickOutputDevice() { Label = "AP Mode - IAS",                Name = "AP.ias",                Byte = 1, Bit = 6 });
-            Lights.Add(new JoystickOutputDevice() { Label = "AP Mode - On/Off",             Name = "AP.autopilot",          Byte = 1, Bit = 7 });
+            Lights.Add(new JoystickOutputDevice() { Label = "AP Mode - HDG", Name = "AP.hdg", Byte = 1, Bit = 0 });
```

Comment:
This really should be a loop that goes over an array of objects and adds them... but you don't have to do it with this PR. When I convert to reading from a JSON file the JSON file will have an array of these JoystickOutputDevice objects in it, I'll read it in, and then loop across them to add it.

Reply:
that's what i also thought!